index.bs

```diff
@@ -671,14 +697,15 @@ dictionary NavigatorUABrandVersion {
 };
 
 dictionary UADataValues {
-  sequence&lt;NavigatorUABrandVersion&gt; brands;
+  FrozenArray&lt;NavigatorUABrandVersion&gt; brands;
```

Comment:
Does that mean that we need to test identity in implementations of the references returned by multiple calls to getHighEntropyValues? Will the Chromium implementation need to align here?